### ISSUE TITLE
Examples to review assistance text for brevity and clarity

### ISSUE THREAD

**Issue by bpdesigns** [18F/e-QIP-prototype#910]
### if so it should be consistent. Certain messages only contain a heading
<img width="633" alt="screen shot 2018-09-28 at 2 07 44 pm" src="https://user-images.githubusercontent.com/15329140/46225785-4e9fbe00-c328-11e8-9ff7-2d2a360091e4.png">

### the examples below are messages where it would be less content to read without diminishing the information being conveyed
<img width="635" alt="screen shot 2018-09-28 at 2 06 58 pm" src="https://user-images.githubusercontent.com/15329140/46225664-f8cb1600-c327-11e8-94e8-2049265b8e4a.png">

_Weight_
**Use pounds and whole numbers only**
Weight must be a number between 10 and 999.

_Hair color_
**Select your current hair color or style**
Choose the hair color that most closely represents your hair today.

_eye color_
**Select your natural eye color**
Choose the eye color that most closely represents your eyes.

_Add a comment_
**Add a comment if needed**
If additional information is appropriate please use "Add a comment" below to detail it.

**Comment by bpdesigns** [comment]
See - [Gov.UK Design System guidance on error messages](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) and w3.orgs [Error messages should be text descriptions that identify](https://www.w3.org/WAI/EO/Drafts/tutorials/forms/error-messages/#error-messages-should-be-text-descriptions-that-identify)
